packages/web-frontend/src/components/View/ChartWrapper/ReferenceLabel.js

```diff
@@ -12,11 +12,13 @@ function ReferenceLabel(props) {
   const x = viewBox.width / 2 + 30;
   const y = viewBox.y + 15;
 
+  if(typeof value != 'undefined' && value !=null) {
```

Comment:
Perhaps it would be worth spending some time getting your linter working? Couple spacing issues here, but the big one is that `!= null` will also catch `undefined`, so there's no need for the first check.
![image](https://user-images.githubusercontent.com/32168886/100160163-28273e80-2f03-11eb-9ed4-eb8c62ef4aef.png)

Reply:
```suggestion
  if(value == null) return null;
```
Also, I believe that this is more clear, we
a) Avoid a level of nesting 
b) Make sure to return something no matter what path the code takes
c) Match a pattern which is used all over the frontend. `if(x) return null;`